src/Hash.mo

```diff
@@ -22,29 +22,4 @@ module {
     ha == hb
   };
 
-  /// Computes a hash from the least significant 32-bits of `n`, ignoring other bits.
```

Comment:
If we aren't going to provide and hash based datastructures (are we?) I'd be tempted to get rid of this whole module Hash. length seemed misnamed (should be maxBitIndex or something) and `bit` is probably an implementation detail of Trie.mo which uses the hash bits to index into a radix tree (IIRC). Might be worth checking if pos/length are used anywhere else.

Reply:
I'll open a PR for this, since this seems reasonable.